source/tyr/tyr/tasks.py

```diff
@@ -235,6 +235,25 @@ def import_autocomplete(files, autocomplete_instance, async=True, backup_file=Tr
 
 
 @celery.task()
+def import_in_mimir(files, instance, async=True):
+    """
+    Import pt data stops to autocomplete
+    """
+    current_app.logger.debug("Import pt data to mimir")
+    actions = []
+    instance_config = load_instance_config(instance.name)
+
+    for _file in files:
+        filename = _file
```

Comment:
you can't do this, you need to pass only one file as argument of import_in_mimir